packages/contracts/test/contracts/L1/messaging/deposit.gas.spec.ts

```diff
@@ -0,0 +1,182 @@
+/* External Imports */
+import { ethers } from 'hardhat'
+import { Signer, ContractFactory, Contract, constants } from 'ethers'
+import { smoddit } from '@eth-optimism/smock'
```

Comment:
Should we be migrating to `@defi-wonderland/smock`?

Reply:
Oh, I wasn't aware of any such package.

Reply:
I created an issue to track that.

Reply:
Kelvin said not yet. 
1. the smock api has changed
2. smoddit is not included in the defi-wonderland package